### ISSUE TITLE
Try run in linux

### ISSUE THREAD

**Issue by maljos** [CreativeDesigner3D/BlenderSource#3]
I download your addons PyClone, Library_Starter and Library_Toy_Box but when I try to add an object to the library it gives me the following error message:

FileNotFoundError: [Errno 2] No existe el archivo o el directorio: '/media/app3d/blender-2.83.5-linux64/blender "/home/mako/.config/blender/2.83/scripts/addons/Library_Toy_Box-master/thumbnail.blend" -b --python "/tmp/blender_L4if6K/thumb_temp.py"': '/media/app3d/blender-2.83.5-linux64/blender "/home/mako/.config/blender/2.83/scripts/addons/Library_Toy_Box-master/thumbnail.blend" -b --python "/tmp/blender_L4if6K/thumb_temp.py"'

location: <unknown location>:-1

Could I ask you please for any suggestions?

**Comment by CreativeDesigner3D** [comment]
Sorry I didn't see this earlier. Are you still having trouble with the new 2.9 release.

**Comment by erictse1981** [comment]
yes,building linux is ok.Drag the wall to build in the screen is ok,But fail at drag the cabinet and appliance
